### ISSUE TITLE
Upload xlsx and docx files in filelist field ends with HTTP 415 Unsupported Media Type

### ISSUE THREAD

**Issue by lexislav** [bolt/core#2608]
I have run into a problem with uploading xls, xlsx, doc, docx files.
These type of files are allowed, but getting this error when trying to upload:

```
UnsupportedMediaTypeHttpException
HTTP 415 Unsupported Media Type
docx files are not accepted
```

The error seems to occur on all environments I have tried.

Details
-------

| Question                      | Answer
|----------------- | ------
| Relevant Bolt Version | master 5.0.0 - beta7 (and beta6 as well)
| Install type                   |  Composer install
| BC Break                     | yes/no
| PHP version                 | 7.4 |
| Web server                  | Apache

**Comment by bobdenotter** [comment]
> These type of files are allowed

Are you sure? How did you configure the setting?

**Comment by lexislav** [comment]
My settings are mostly untouched from originals.

No changes made in config.yaml file types at all.
In content type the field definition looks like this:

```
files:
    type: filelist
     group: Downloads
     #extensions: [ doc, docx, pdf, xls, xlsx ] #I have turned off the file types restriction right now
     upload: downloads
```

So until I have not missed some other settings about the type files, I am sure…

PDF files are uploading without problems, images as well in imagelist.
The error shows for doc, docx, xls, xlsx, zip files.

Error screenshot from two servers (one is set to dev, one to production, so the error show differently)

![image](https://user-images.githubusercontent.com/7147349/120606958-024d9a00-c450-11eb-86bd-8ae49becd1cb.png)
![image](https://user-images.githubusercontent.com/7147349/120607312-5bb5c900-c450-11eb-9d85-277b283185a9.png)

**Comment by lexislav** [comment]
Fixed on 5.0.0-beta.12, thx.
